src/python/grpcio/grpc/experimental/__init__.py

```diff
@@ -85,6 +85,6 @@ def _wrapper(*args, **kwargs):
     'insecure_channel_credentials',
 )
 
-if sys.version_info[0] >= 3:
+if sys.version_info[0] == 3 and sys.version_info[1] >= 6:
```

Comment:
If you remove the f-string, then we can support 3.6 users. Simple stubs can be available to more users.

Reply:
True. This was originally a middle-of-the-night quick fix. Now that the pressure is off, I'll take some time to try to extend support back to 3.5.

Reply:
Sure. Claim supporting 3.5 might need extra efforts.

Reply:
Lidi, I took another look at this and the problem is more than just f-strings. It's also [PEP 526 variable annotations](https://www.python.org/dev/peps/pep-0526/) that don't work in 3.5. I think it's reasonable to have the same interpreter requirements as our `asyncio` support, namely, 3.6+.